tests/scripts/download_gherkin_features.bat

```diff
@@ -0,0 +1,6 @@
+mkdir .\tests\tempFeatures
```

Comment:
Will it be possible to add some docs/headers about how this particular script works, what arguments are required and what requirements are needed?